FTBFS due to MOUSE_* not being defined with FEAT_SMALL

--- origsrc/vim73/src/feature.h	2013-05-13 12:58:34.639790600 -0500
+++ src/vim73/src/feature.h	2013-05-13 15:27:48.418748300 -0500
@@ -1137,7 +1137,7 @@
  *			in an xterm like in the GUI.
  */
 
-#ifdef FEAT_CYGWIN_WIN32_CLIPBOARD
+#if defined(FEAT_CYGWIN_WIN32_CLIPBOARD) && defined(FEAT_NORMAL)
 # define FEAT_CLIPBOARD
 #endif
 
--- origsrc/vim73/src/os_unix.c	2013-05-13 12:58:22.159511200 -0500
+++ src/vim73/src/os_unix.c	2013-05-13 15:28:11.656077400 -0500
@@ -1274,7 +1274,7 @@ mch_init()
 #ifdef MACOS_CONVERT
     mac_conv_init();
 #endif
-#ifdef FEAT_CYGWIN_WIN32_CLIPBOARD
+#if defined(FEAT_CYGWIN_WIN32_CLIPBOARD) && defined(FEAT_NORMAL)
     win_clip_init();
 #endif
 }
